doc/CORTX_MOTR_ARCHITECTURE.rst

```diff
@@ -0,0 +1,600 @@
+.. vim: syntax=rst
+
+|image0|
+
+
+
+============
+**Outline**
+============
+
+-  This presentation is a beginner's guide to the CORTX core architecture
+
+-  Tries to cover all important aspects of CORTX core with a reasonable
+   level of detail
+
+-  Focus on software (hardware details elsewhere)
+
+
+
+
+-  Overview
+
+-  Typical use cases and data flow
+
+-  How objects and indices are stored
+
+-  IO path
+
+-  DTM
+
+-  FDMI
+
+-  ADDB
+
+
+=================
+**What is CORTX?**
+=================
+
+
+-  CORTX core is a component of the new cloud software stack developed by Seagate
+
+-  CORTX provides basic functionality through its clovis interface:
```

Comment:
Need to remove clovis string.

Reply:
You are right.

Original doc is needed to edit the image. Otherwise, it will be time-consuming.

Reply:
The original doc:

https://seagatetechnology.sharepoint.com/:p:/r/sites/GMSites/edsengineering/_layouts/15/Doc.aspx?sourcedoc=%7B6D890421-A1B3-400F-BDE1-726E5FD1CB01%7D&file=EOS%20core%20architecture%20overview.pptx&action=edit&mobileredirect=true&DefaultItemOpen=1

Reply:
@johnbent , @nikitadanilov do you think we need to update all the images in the doc to reflect the name changes, e.g.  from Clovis to Motr client?

Reply:
Anyway, it is fixed now.

Reply:
@imvenkip would you like to take another look?

Reply:
Sure.

Reply:
Looks good to me.